examples/CreateSFMExampleData.cpp

```diff
@@ -63,49 +57,68 @@ void createExampleBALFile(const string& filename, const vector<Point3>& P,
 /* ************************************************************************* */
 
 void create5PointExample1() {
-
   // Create two cameras poses
   Rot3 aRb = Rot3::Yaw(M_PI_2);
   Point3 aTb(0.1, 0, 0);
   Pose3 pose1, pose2(aRb, aTb);
 
   // Create test data, we need at least 5 points
   vector<Point3> P;
-  P += Point3(0, 0, 1), Point3(-0.1, 0, 1), Point3(0.1, 0, 1), //
-  Point3(0, 0.5, 0.5), Point3(0, -0.5, 0.5);
+  P += Point3(0, 0, 1), Point3(-0.1, 0, 1), Point3(0.1, 0, 1),  //
+      Point3(0, 0.5, 0.5), Point3(0, -0.5, 0.5);
 
   // Assumes example is run in ${GTSAM_TOP}/build/examples
-  const string filename = "../../examples/data/5pointExample1.txt";
+  const string filename = "../../examples/Data/5pointExample1.txt";
   createExampleBALFile(filename, P, pose1, pose2);
 }
 
 /* ************************************************************************* */
 
 void create5PointExample2() {
-
   // Create two cameras poses
   Rot3 aRb = Rot3::Yaw(M_PI_2);
   Point3 aTb(10, 0, 0);
   Pose3 pose1, pose2(aRb, aTb);
 
   // Create test data, we need at least 5 points
   vector<Point3> P;
-  P += Point3(0, 0, 100), Point3(-10, 0, 100), Point3(10, 0, 100), //
-  Point3(0, 50, 50), Point3(0, -50, 50), Point3(-20, 0, 80), Point3(20, -50, 80);
+  P += Point3(0, 0, 100), Point3(-10, 0, 100), Point3(10, 0, 100),  //
+      Point3(0, 50, 50), Point3(0, -50, 50), Point3(-20, 0, 80),
+      Point3(20, -50, 80);
 
   // Assumes example is run in ${GTSAM_TOP}/build/examples
-  const string filename = "../../examples/data/5pointExample2.txt";
+  const string filename = "../../examples/Data/5pointExample2.txt";
```

Comment:
why uppercase Data now, instead of lowercase data?

Reply:
The folder in gtsam is now called "Data". I believe this file is a bit old.